pkg/controller/quota/quota.go

```diff
@@ -193,20 +193,23 @@ func addStorage(appInstance *v1.AppInstance, quotaRequest *adminv1.QuotaRequestI
 	return nil
 }
 
+// defaultVolumeSize determines the default size of the specified volume. If the volume has a default size set
+// on the status.Defaults.Volumes, it uses that. Otherwise, it uses the default size set on the status.Defaults.VolumeSize.
 func defaultVolumeSize(appInstance *v1.AppInstance, name string) resource.Quantity {
+	// Figure out what the default volume size should be. If the status has a default volume size set, use that. Otherwise,
+	// use the default size set in the v1 package.
 	result := *v1.DefaultSize // Safe to dereference because it is statically set in the v1 package.
+	if appInstance.Status.Defaults.VolumeSize != nil {
+		result = *appInstance.Status.Defaults.VolumeSize
```

Comment:
I would like to see this included as part of `appInstance.Status.Defaults.Volumes` instead of it being its own field. Otherwise, this field is going to need to be checked everywhere we are trying to determine the default.

Reply:
That would require some migration of apps going to `Defaults.VolumeSize` => `Defaults.Volumes[volume].Size`. Do you want that to start with this PR?